openedx/features/enterprise_support/api.py

```diff
@@ -241,13 +243,22 @@ def enterprise_customer_for_request(request, tpa_hint=None):
 
     ec = None
 
+    running_pipeline = get_partial_pipeline(request)
+    if running_pipeline:
+        # Determine if the user is in the middle of a third-party auth pipeline,
+        # and set the tpa_hint parameter to match if so.
+        tpa_hint = Registry.get_from_pipeline(running_pipeline).provider_id
```

Comment:
@haikuginger Is "TPA hint" a *thing*/specific concept in the realm of third-party authentication? Or is it something that has a specific meaning in the context of ENT code? I'm asking because looking at this code it seems like `provider_id` would be appropriate to use as a variable name (and a bit more to the point).